We need to readlink argv[0] to get the correct prefix of gpodder

Signed-of-by: Jonas Karlsson <jonas@gobolinux.org>
--- gpodder-0.11.0/bin/gpodder	2008-02-25 12:49:43.000000000 +0100
+++ gpodder-0.11.0/bin/gpodder.new	2008-03-24 14:09:49.000000000 +0100
@@ -53,7 +53,7 @@
 
 
 def main( argv = sys.argv):
-    prefix = os.path.abspath( os.path.normpath( os.path.join( os.path.dirname( argv[0]), '..')))
+    prefix = os.path.abspath( os.path.normpath( os.path.join( os.path.dirname( os.readlink(argv[0])), '..')))
     locale_dir = os.path.join( prefix, 'share', 'locale')
 
     # Enable i18n support
